### PR TITLE
Add ModalCloseButton

### PR DESCRIPTION
**Pull request by hizegi**
Testing hilary's changes, just in a different branch

### CODE REVIEW

File: lib/main.js [hunk 1]
```diff
@@ -45,6 +45,7 @@ export {
   ListIcon,
   ListItem,
   Modal,
+  ModalCloseButton,
```
Comment (leskat47) [inline lib/main.js:48]: Is this related?

Comment (hizegi) [reply lib/main.js:48]: Related in what sense? This is just a copy of Hilary's work she's been trying to add this `ModalCloseButton` 😆

Comment (hilaryBeck) [reply lib/main.js:48]: yeah this wasn't what caused the error, just the one little change i was trying to make lol

**Review by hilaryBeck — APPROVED** [review]
